apps/mobile/src/screens/welcome-screen/welcome-screen.tsx

```diff
@@ -1,12 +1,59 @@
 import React from "react";
 
-import { Text, View } from "~/libs/components/components";
+import {
+	BackgroundWrapper,
+	Button,
+	Image,
+	ScreenWrapper,
+	Text,
+	View,
+} from "~/libs/components/components";
+import { globalStyles } from "~/libs/styles/styles";
+import { ImageSourcePropType } from "~/libs/types/types";
+
+import { styles } from "./styles";
+
+const handleContinuePress = (): void => {
+	// Handle Continue
+};
 
```

Comment:
Should be inside of component

Reply:
`[js]   17:39  error  Move arrow function 'handleContinuePress' to the outer scope  unicorn/consistent-function-scoping`